src/app/components/tree-view/index.html

```diff
@@ -126,20 +126,20 @@ <h3>Components</h3>
       sectionHeading="Angular tree view components"
     >
       <p>
-        To apply the SKY UX themes and additional functionality to the <sky-code>angular-tree-component</sky-code>, you must include the following components.
+        <sky-code>SkyAngularTreeModule</sky-code> provides SKY UX components and styles to complement the Angular tree component and apply SKY UX themes and functionality to hierarchical list views. For information about the Angular tree component and its capabilities, see the <a href="https://angular2-tree.readme.io/docs">Angular tree component documentation</a>.
```

Comment:
Is “angular-tree-component” the actual selector for the Angular component? I can’t tell from there docs, but that doesn’t seem right. I see in some of there reference docs that they have it presented this way (but not as a code snippet), but I see other places where the refer to it generically. I changed all of our references like this with the code snippet formatting to the generic “Angular tree component” with no code snippet formatting because it didn't seem right. But since I’m not sure if that’s correct, I just wanted to mention that I made this change throughout..

Reply:
I'm going to resolve this comment based on Alex's comments above on the name of the module.

Reply:
That's probably for the better. You're right - there is no `angular-tree-component` selector. I should have referred to the generic component like you did above. 👍